.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,26 @@
+name: automerge
+on:
+  pull_request:
```

Comment:
Does this need to be limited to the `develop` branch?

Reply:
it only cares about the label

Reply:
right...I just didn't know if that was an extra restriction we wanted to put in or if we don't care. (I suspect the later)